roles/github/README.md

```diff
@@ -57,62 +60,57 @@ If you wish to make more impactful changes such as which workflows are built and
 
 - `arguments`: list of arguments keys used by the automation task the contents will be acquired from `kayobe_arguments` or the defaults.
 
-The following will override `workflows` to ensure only `Run overcloud database backup` is generated.
+- `use_bespoke`: Some workflows benefit from a dedicated workflow template as they drift away from the main template. Set to `true` if the workflow requires a *bespoke* template and ensure a template `workflow_name.yml.j2` is present.
+
+The following will override `workflows` to ensure only `Build Kayobe Image` and `Run Kolla Config Diff` is generated.
 
 ```yaml
-workflows:
-  - name: Run overcloud database backup
-    file_name: run-overcloud-database-backup.yml
-    trigger:
-      workflow_dispatch: *combined_inputs
-      schedule:
-        cron: "30 0 * * *"
-    arguments:
-      - KOLLA_TAGS
-      - KOLLA_LIMIT
-      - KAYOBE_TAGS
-      - KAYOBE_LIMIT
-      - HOME
+github_workflows:
+  - "{{ build_kayobe_image }}"
+  - "{{ run_kolla_config_diff }}"
 ```
 
-Example Playbook
-----------------
+Template Hooks
+--------------
 
-The following example playbook will generate a series of `reference` workflows which can be found under `.github/workflows`
+Workflows can be expanded with the use of hooks which are variables that if provided can be inserted into the appropriate location enabling the introduction of additional steps within the workflow job. This could include the use of HashiCorp Vault or installing and configuring a network proxy.
 
-```yaml
-- name: Write Kayobe Automation Workflows
-  hosts: localhost
-  collections:
-    - stackhpc.kayobe_automation_workflows
-```
-
-Hooks [Experimental]
---------------------
+There are currently three hooks available
 
-> :warning: This feature is marked as `experimental` at the moment as it is not clear how to configure `ansible.builtin.template` to look at the `${{ playbook_dir }}/templates/hooks/`.
+- `github_checkout_hook`: a hook that occurs before the repository is cloned by the `checkout` action.
```

Comment:
github_pre_checkout_hook?

Reply:
I have omitted the `pre` qualifier as I had removed the `post` hooks entirely which felt redundant.

The idea is each hook occurs before the named event. However, I can see the benefit in being clearer and can `pre` back.